### PR TITLE
docs(tutorial): Writing Custom Configuration Audit Policies

### PR DESCRIPTION
**Pull request by danielpacak**
To preview docs on this branch run `make mkdocs-serve` and then open [http://localhost:8000](http://localhost:8000) in your browser.

Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

### CODE REVIEW

File: docs/tutorials/writing_custom_configaudit_policies.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    	provided := {label | input.metadata.labels[label]}
+    	required := {label | label := recommended_labels[_]}
+    	missing := required - provided
+    	count(missing) > 0
+    	msg := sprintf("You must provide labels: %v", [missing])
+    	res := {"msg": msg}
+    }
+```
+
+As you can see in this example, to add a new policy you must define two data entries in the `starboard-policies-config`
```
Comment (chen-keinan) [inline docs/tutorials/writing_custom_configaudit_policies.md:137]:
```suggestion
In this example, to add a new policy, you must define two data entries in the `starboard-policies-config`
```

**Review by chen-keinan — CHANGES_REQUESTED** [review]

File: docs/tutorials/writing_custom_configaudit_policies.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    	"app.kubernetes.io/version",
+    ]
+
+    deny[res] {
+    	provided := {label | input.metadata.labels[label]}
+    	required := {label | label := recommended_labels[_]}
+    	missing := required - provided
+    	count(missing) > 0
+    	msg := sprintf("You must provide labels: %v", [missing])
+    	res := {"msg": msg}
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/tutorials/writing_custom_configaudit_policies.md:133]: We always evaluate the `__rego_metadata__` rule so it's not required to do what you suggested.

File: docs/tutorials/writing_custom_configaudit_policies.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
+# Writing Custom Configuration Audit Policies
+
+Starboard ships with a set of pre-installed configuration audit policies defined as OPA [Rego] policies. You can also
```
Comment (AnaisUrlichs) [inline docs/tutorials/writing_custom_configaudit_policies.md:3]: Where can people learn more about our pre-installed configuration audit policies? Would be worth linking

File: docs/tutorials/writing_custom_configaudit_policies.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+      {
+        "name": "nginx",
+        "image": "nginx:1.16",
+      }
+    ]
+  }
+}
+```
+
+Therefore, provided labels can be retrieved with the following Rego expression:
```
Comment (AnaisUrlichs) [inline docs/tutorials/writing_custom_configaudit_policies.md:82]: I would make it more explicit -- "The labels set on the pod resource above can be retrieved with the following Rego expression:"

File: docs/tutorials/writing_custom_configaudit_policies.md (hunk 3 above)
Comment (danielpacak) [reply docs/tutorials/writing_custom_configaudit_policies.md:3]: There's no such place, but I'm going to list built-in policies in scope of #1019. Then we could update this tutorial with reference.

File: docs/tutorials/writing_custom_configaudit_policies.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chen-keinan) [reply docs/tutorials/writing_custom_configaudit_policies.md:133]: Its just less compatible with how we write k8s rego policy in defsec repo

[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/tutorials/writing_custom_configaudit_policies.md:133]: We are just reading the msg property anyway.

File: docs/tutorials/writing_custom_configaudit_policies.md (hunk 4 above)
Comment (danielpacak) [reply docs/tutorials/writing_custom_configaudit_policies.md]: Done

**Review by chen-keinan — APPROVED** [review]

**Pull request merged**